modular_citadel/code/modules/reagents/chemistry/reagents/healing.dm

```diff
@@ -0,0 +1,88 @@
+/datum/reagent/fermi/yamerol
+	name = "Yamerol"
+	id = "yamerol"
+	description = "For when you've trouble speaking or breathing, just yell YAMEROL! A chem that helps soothe any congestion problems and at high concentrations restores damaged lungs and tongues!"
+	taste_description = "a weird, syrupy flavour, yamero"
+	color = "#68e83a"
+	pH = 8.6
+	overdose_threshold = 35
+	ImpureChem 			= "yamerol_tox" //If you make an inpure chem, it stalls growth
+	InverseChemVal 		= 0.3
+	InverseChem 		= "yamerol_tox" //At really impure vols, it just becomes 100% inverse
+
+/datum/reagent/fermi/yamerol/on_mob_life(mob/living/carbon/C)
+	var/obj/item/organ/tongue/T = C.getorganslot(ORGAN_SLOT_TONGUE)
+	var/obj/item/organ/lungs/L = C.getorganslot(ORGAN_SLOT_LUNGS)
+	message_admins("Yamero!")
```

Comment:
You are hopping for troubles by carelessly pushing message_admins on this TM'd PRs like this, mister.
Remove.